app/i18n/de/admin.php

```diff
@@ -122,7 +122,7 @@
 		'title' => 'Erweiterungen',
 		'update' => 'Update verfügbar',
 		'user' => 'Benutzer-Erweiterungen',
-		'version' => 'Version',	// TODO - Translation
+		'version' => 'Version',
```

Comment:
Did you also add it to the ignore list? :-)

Reply:
Not yet, I'll update the ignore list.